src/test/perl/test-caffileeditor_source.t

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/perl
+use strict;
+use warnings;
+use FindBin qw($Bin);
+use lib "$Bin/modules";
```

Comment:
this is what loads dummy lc modules, remove it and you should be able to use `LC::File::file_contents` without the mocking